packages/render-fidelity-tools/src/common.ts

```diff
@@ -226,20 +226,26 @@ export class ImageComparator {
           candidateImage.length}, golden: ${goldenImage.length})`);
     }
 
+    let modelPixelCount = 0;
     for (let y = 0; y < height; ++y) {
       for (let x = 0; x < width; ++x) {
         const index = y * width + x;
         const position = index * COMPONENTS_PER_PIXEL;
+
+        if (candidateImage[position + 3] == 0) {
```

Comment:
This is a good candidate for a comment; it may not be obvious to most readers that you are ignoring background pixels by looking for alpha = 0.